scripts/setup/run_build_tool.sh

```diff
@@ -33,7 +33,7 @@ done
 
 if [[ $ENABLE_SCCACHE == "true" ]]; then
 	env | grep -E "^SCCACHE_" >"${CARGO_HOME}/sccache.env"
-	EXTRA_ARGS="${EXTRA_ARGS} --env RUSTC_WRAPPER=/opt/rust/cargo/bin/sccache --env-file ${CARGO_HOME}/sccache.env"
+	EXTRA_ARGS="${EXTRA_ARGS} --env RUSTC_WRAPPER=/opt/rust/cargo/bin/sccache --env CARGO_INCREMENTAL=0 --env-file ${CARGO_HOME}/sccache.env"
```

Comment:
Maybe better to set this in https://github.com/datafuselabs/databend/blob/main/.github/actions/build_linux/action.yml#L19?

Reply:
https://github.com/datafuselabs/databend/blob/72d3b00bc66cb1a123405f31aec3878faa61066a/.github/workflows/dev-linux.yml#L19-L21

What about setting it here? I think we can actually disable incremental build regardless of sccache.

Reply:
> Disable incremental compilation. CI builds often are closer to from-scratch builds, as changes are typically much bigger than from a local edit-compile cycle. For from-scratch builds, incremental adds an extra dependency-tracking overhead. It also significantly increases the amount of IO and the size of ./target, which make caching less effective.

https://matklad.github.io/2021/09/04/fast-rust-builds.html

Reply:
LGTM!